scripts/validate_rst_title_capitalization.py

```diff
@@ -141,6 +141,12 @@
     "False",
     "Styler",
     "os",
+    "Cython"
+    "Numba",
+    "MultiIndexes",
+    "CSV",
+    "OpenDocument",
+    "msgpack"
```

Comment:
Probably need to add a trailing comma here, running black on this should take care of any formatting issues